src/modules/holder/pages/Home/Home.tsx

```diff
@@ -69,34 +66,36 @@ export const Home: FC = () => {
   const getCredential = (credentialSubject: AnyData, credentialItem: StoredW3CCredential) => {
     const credential: Credential = {
       title: credentialSubject?.eventName,
-      date: formatDate(new Date(credentialSubject?.startDate).toISOString().substring(0, 10)),
+      date: format(new Date(credentialSubject?.startDate), 'dd.MM.yyyy'),
       time: new Date(credentialSubject?.startDate).toISOString().substring(11, 16),
       credentialId: credentialItem?.id,
     }
 
     return credential
   }
 
-  const getTicketCards = (array: StoredW3CCredential[], object: { isValid: boolean }) =>
-    array.map((credentialItem: StoredW3CCredential) => {
+  const getTicketCards = (tickets: { ticketsArray: StoredW3CCredential[]; isValid: boolean }) =>
+    tickets.ticketsArray.map((credentialItem: StoredW3CCredential) => {
       const credentialSubject = credentialItem?.credentialSubject
 
       const credential = getCredential(credentialSubject, credentialItem)
-      return <TicketCard key={credentialItem.id} credential={credential} isValid={object.isValid} />
+      return (
+        <TicketCard key={credentialItem.id} credential={credential} isValid={tickets.isValid} />
+      )
     })
 
   return (
     <>
       <Header title="Your tickets" />
 
       <Container isGrid>
-        {validTickets && getTicketCards(validTickets, { isValid: true })}
+        {validTickets && getTicketCards({ ticketsArray: validTickets, isValid: true })}
       </Container>
 
       {expiredTickets.length !== 0 && <Header title="Expired tickets" />}
 
       <Container isGrid>
-        {expiredTickets && getTicketCards(expiredTickets, { isValid: false })}
+        {expiredTickets && getTicketCards({ ticketsArray: expiredTickets, isValid: false })}
```

Comment:
no need to add an array suffix because it is redundant 
when u use plural it automatically means an array
```suggestion
        {expiredTickets && getTicketCards({ tickets: expiredTickets, isValid: false })}
```